### DISCUSSION TITLE
command 'rmarkdown.rmarkdown-core.Knit' not found!

### DISCUSSION THREAD

**Discussion by bigdatalib** [quarto-dev/quarto-cli#1550] [Q&A] [answered]
Hi All, 
I just installed quatro on my Windows machine. I am using VS Code. When trying to render (ctr+shift+k) I am getting the following error: command 'rmarkdown.rmarkdown-core.Knit' not found. Have you seen this before? Do you know how to fix it?

**Comment by cscheid** [comment, marked as answer]
It looks like VS code thinks you're attempting to render a RMarkdown file. If you want to use Quarto in VS Code, you should make sure you're using the Quarto extension and that it's enabled for your file. Have you gone over the instructions here? https://quarto.org/docs/get-started/hello/vscode.html

**Reply by bigdatalib** [reply]
Thank you very much. It seems I had an extension in vs code called: R Markdown All in one. I had to disable that.... now it works! thank you for your help.
